streams/do.m3u

```diff
@@ -295,3 +295,5 @@ https://ss3.domint.net:3108/zol_str/vzol/playlist.m3u8
 https://lb00zdigital.streamprolive.com/mnt/hls/live.m3u8
 #EXTINF:-1 tvg-id="Area809ElOriginal.do",Area 809 El Original (1080p)
 https://samson.streamerr.co:8081/rudydurisimotv/index.m3u8
+#EXTINF:-1 tvg-id="LunaTV.do",LunaTV (1080p)
```

Comment:
```suggestion
#EXTINF:-1 tvg-id="",LunaTV (1080p)
```

There is no such channel in the [iptv-org/database](https://github.com/iptv-org/database).

https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#stream-description-scheme